src/components/Vuetify/mixins/breakpoint.ts

```diff
@@ -111,8 +83,16 @@ export default Vue.extend({
       this.resizeTimeout = window.setTimeout(this.setDimensions, 200)
     },
     setDimensions (): void {
-      this.clientHeight = getClientHeight()
-      this.clientWidth = getClientWidth()
+      const width = getClientWidth()
```

Comment:
this method (after small tweaking) could be used in `data()` to avoid code duplication

Reply:
I think I should unify getClientWidth and getClientHeight to return a tuple, they are always run together. But I am not sure if destructuring the tuple is actually worse for performance than two function calls :man_shrugging:
Maybe it should return an object so that the names `height` and `width` could be put into those two contexts by destructuring? :woman_shrugging:

Reply:
I could return an object of
```ts
clientHeight: height,
clientWidth: width,
width,
height,
cl,
...breakpointExplainations[cl]
```
but then I'd have to merge that in with 
```ts
{resizeTimeout: undefined as number | undefined}
```
and in the second case, simply assign it to
```ts
Object.assign(this, extractedFunction())
```
So there would be two destructurings/calls to Object.assign per method (`data` and `setDimensions`) Since both of those already have a call like that, I used those.

The only reason a second Object.assign is used in the `cl` watcher is because many resizes might not end up changing the breakpoint, meaning we can skip assigning all of those numerous keys (and Vue individually comparing them all)